pydash/arrays.py

```diff
@@ -1813,6 +1912,9 @@ def iterunique(array, comparator=None, iteratee=None):
             seen.append(cmp_item)
 
 
+iterunion = iterunique
```

Comment:
My other comment says to use the uniq_by and uniq_with functions for union_by/with respectively so there's no reason for this function. I'm not liking to alias helper functions that are only used internally. That just adds unnecessary extras that serve little purpose.